src/dimension_slice.h

```diff
@@ -53,7 +53,8 @@ extern DimensionVec *ts_dimension_slice_collision_scan_limit(int32 dimension_id,
 extern bool ts_dimension_slice_scan_for_existing(DimensionSlice *slice);
 extern DimensionSlice *ts_dimension_slice_scan_by_id_and_lock(int32 dimension_slice_id,
 															  ScanTupLock *tuplock,
-															  MemoryContext mctx);
+															  MemoryContext mctx)
+	TS_ATTRIBUTE_NONNULL((2));
```

Comment:
Should the lines be identified on the same level?

Reply:
This is the result of running `clang-format`, so doing it another way will fails some jobs.

Reply:
I think you are right. This macro is really confusing :)